charts/rancher-monitoring-crd/105.1.0-rc1+up61.3.2/README.md

```diff
@@ -0,0 +1,24 @@
+# rancher-monitoring-crd
+A Rancher chart that installs the CRDs used by rancher-monitoring.
```

Comment:
So I see this here - but in Rancher under "Installed Apps" when I go to monitoring and into the "Chart README" tab it's the upstream readme. 🤔

Reply:
There shouldn't be a readme for the upstream crd. I don't think you can even view the crd chart README without some shenanigans. Can you confirm that you're looking at the README for `rancher-monitoirng` or `rancher-monitoring-crd`?

Reply:
I think there's something going on - since the source of the readme I'm seeing in the UI matches this: https://github.com/rancher/charts/tree/dev-v2.9/charts/rancher-monitoring/104.1.1-rc1%2Bup57.0.3

(obviously versions vary, but this looks like a long standing thing)